test/GithubApi.Redirect.js

```diff
@@ -0,0 +1,35 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const { expect } = chai;
+const urlBase = 'https://github.com/aperdomob';
+
+describe('when we have a redirect', () => {
+  let redirectResponse;
+  const redirectURL = `${urlBase}/redirect-test`;
+  const urlAfterRedirect = `${urlBase}/new-redirect-test`;
+
+  before(() => agent.head(`${redirectURL}`)
+    .catch((error) => {
+      redirectResponse = error;
+    }));
+
+  it('then it should redirect', () => {
```

Comment:
Este paso no es lo que estás validando, debería ser un status should be 301 o algo asi